packages/terra-button/src/Button.jsx

```diff
@@ -200,7 +202,7 @@ class Button extends React.Component {
     // Prevent button from showing focus styles when clicked
     this.shouldShowFocus = false;
     // Wait until after onFocus has been triggered on browsers where it will get triggered for click
-    setTimeout(() => { this.shouldShowFocus = true; }, 300);
+    setTimeout(() => { this.shouldShowFocus = true; });
```

Comment:
The zero second timeout has been added and removed a couple times. Can you explain why it is needed?

Reply:
@StephenEsser I tried defending not to include. But @benbcai had other suggestions.

You can refer to this [comment](https://github.com/cerner/terra-core/pull/2861#discussion_r390464106) from above.

We need to wait until what @benbcai has to say on this.